Daytime-Operations/Safety/Control-Safety-Systems.rst

```diff
@@ -0,0 +1,112 @@
+.. This is a template for operational procedures. Each procedure will have its own sub-directory. This comment may be deleted when the template is copied to the destination.
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Giovanni Corvetto*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Daytime-Operations-Safety-Control-Safety-Systems:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################
+Control Safety Systems
+######################
+
+Presented here is an overview of the concepts LOTO, GIS, and E-STOP, and when each procedure or system is used. 
+Specific procedures, like how to LOTO the Telescope Mount Assembly (TMA), or how to reset the GIS to resume operations of a specific component, are not described here. 
+The final procedures for LOTO will be documented on the Observatory Operations pages once they are approved by the Safety Team, or will require hands-on training. 
```

Comment:
Do you plan to link them from here?

Reply:
I planned to keep this page informational. What do you recommend?

Reply:
If the page exists then we should link it... 
so I suppose either add the link or say it'll be linked it in the future...

Reply:
Specific procedures for LOTO aren't written down yet because they haven't reached their final form. I will link them here. 👍